channeldb/invoice_test.go

```diff
@@ -432,7 +452,7 @@ func TestFetchAllInvoicesWithPaymentHash(t *testing.T) {
 			t.Fatalf("unable to create invoice: %v", err)
 		}
 
-		invoice.State = states[mrand.Intn(len(states))]
+		invoice.State = states[int(i)%len(states)]
```

Comment:
This could have been a separate commit just to get rid of the random.

Reply:
done